examples/bindgen/simple.h

```diff
@@ -1,3 +1,12 @@
 #include <stdint.h>
 
+#if defined(__APPLE__)
+// https://github.com/bazelbuild/rules_rust/issues/899
+#include <TargetConditionals.h>
+
+const int8_t IS_MACOS = TARGET_OS_OSX;
+#else
+const int8_t IS_MACOS = 0;
```

Comment:
Could you also add a test that uses this value in the `main.rs` file? Where in `#[cfg(target_os = "macos")]` would test that the value is maybe not some invalid default (`-1`?)